source/install/config-tls-mattermost.rst

```diff
@@ -42,3 +42,17 @@ The easiest option is to set up TLS on the Mattermost Server, but if you expect
 
 .. note::
    Password-protected certificates are not supported.
+   
+**Configure TLS on NGINX (proxy)**
```

Comment:
This section is called "Configure" but the three steps below seem to be a list of functionality available as opposed to instructions. One option is to rename this header to something like "Using TLS on NGINX as a proxy", with a brief introduction (using point one), followed by points two and three being turned into bullet-points. The more context and detail around this, the better. :)

Reply:
done